retrofit/build.gradle

```diff
@@ -26,3 +26,26 @@ jar {
     attributes  'Automatic-Module-Name': 'retrofit2'
   }
 }
+
+// Create a test task for each supported JDK.
+(8..17).each { majorVersion ->
+  // Adoptium JDK 9 cannot extract on Linux or Mac OS.
+  if (majorVersion == 9) return
```

Comment:
I _think_ this is a Gradle bug, but I'm not sure. I don't really care since it's only one really old JDK.